### ISSUE TITLE
Not working in IE

### ISSUE THREAD

**Issue by schwammy** [Microsoft/BotFramework-WebChat#841]
I've got the webchat working correctly in Chrome but not in IE. 
My app is an Angular website using WebChat to connect to my bot. 

Code looks like this:
_(this is Typescript code, so the arrow functions are transpiled out, I believe. But I tried it with and without them with the same result.)_
``` 
directLine: DirectLine = new DirectLine({ secret: environment.botDirectLineSecretKey }) 
  
@ViewChild("botWindow") botWindowElement: ElementRef;

ngOnInit() {

    this.directLine.activity$
      .filter(activity => activity.type === "event" && activity.name === "init")
      .subscribe(activity => this.changeSize());

    BotChat.App({
      botConnection: this.directLine,
      user: { id: 'user' },
      bot: { id: 'bot' },
    }, this.botWindowElement.nativeElement);

}

changeSize(){
    console.log("here")
    var container = document.getElementById("bot-chat-container");
    container.classList.add("fullSize");
}    
```
In Internet Explorer, I get this error in the console:

> ERROR TypeError: You provided an invalid object where a stream was expected. You can provide an Observable, Promise, Array, or Iterable.

If I switch `botConnection: this.directLine` to `directLine: this.directLine`, the webchat works but then the changeSize() method never gets called.

Any help is appreciated. This bot is going on a public website and IE must be supported.

**Comment by schwammy** [comment]
I resolved this issue myself. After realizing that I could make this work with "vanilla" JavaScript, I figured the problem was somewhere in the TypeScript. I did this:

Changed this: `directLine: DirectLine = new DirectLine({ secret: environment.botDirectLineSecretKey })`

to this: `directLine: DirectLine = new BotChat.DirectLine({ secret: environment.botDirectLineSecretKey })` (note that I am now using **BotChat**.DirectLine)

Everything is working nicely. I'm not really sure why the original works in Chrome but the second works on both Chrome and IE.
